src/errors.ts

```diff
@@ -56,3 +59,7 @@ export function isWSFailure(err: APIError): boolean {
     return false;
   }
 }
+
+export function isErrorResponse(res: AxiosResponse<unknown>): res is AxiosResponse<APIErrorResponse> {
+  return (res.status + '')[0] !== '2';
```

Comment:
this is very clever but isn't `res.status && res.status > 300` easier to read?

Reply:
@peterdeme this was just copied from the original place of use. You are right that it is hacky. I am just wandering, why the author of the one liner included also status 100 among error responses. If I would be translating the original one liner into something more readable, then it would be `!!res.status || res.status < 200 && 300 >= res.status`. Would you be ok with that one?